src/sql/workbench/browser/modelComponents/media/card.css

```diff
@@ -66,8 +66,8 @@
 	flex-grow: 1;
 	width: 100%;
 	height: 100%;
-	max-width: 50px;
-	max-height: 50px;
+	max-width: 100px;
```

Comment:
what happens if you remove the maxwidth and max height properties

Reply:
it looks the same if I remove them. I wasn't sure if I should just increase the max width or remove them. Should I remove it so that people can make their icon as big as they want so they can potentially get into a state like this if they make their icon really big? 
![image](https://user-images.githubusercontent.com/31145923/100681483-af701a00-3328-11eb-95be-2c4d3f6b761d.png)